openembedded-core-mbl/scripts/lib/mbl-signing/signing/pki/tbbr_defs.py

```diff
@@ -84,12 +84,12 @@
         TRUSTED_FW_NVCOUNTER=CounterExtension(
             oid="1.3.6.1.4.1.4128.2100.1", critical=True
         ),
-        APROM_PATCH_HASH=HashExtension(
-            oid="1.3.6.1.4.1.4128.2100.601", critical=False
-        ),
-        SOC_CONFIG_HASH=HashExtension(
-            oid="1.3.6.1.4.1.4128.2100.602", critical=True
-        ),
+        #APROM_PATCH_HASH=HashExtension(
```

Comment:
Presumably commented out as not supported yet?  Can we have a TODO/Future comment or something?

Reply:
Commented out as we don't use it in our trusted boot implementation. Deleted in 139aff9